spec/mailers/provider_mailer_spec.rb

```diff
@@ -149,4 +151,14 @@
       expect(@mail.body.encoded).to include("by #{rbd_date.to_s(:govuk_date).strip}")
     end
   end
+
+  describe '.offer_accepted' do
+    before do
+      @mail = mailer.offer_accepted(provider_user, application_choice)
+    end
+
+    it 'mentions candidate and course' do
```

Comment:
Add assertion about for the course name and code?

Reply:
Done!